libMBIN/Source/NMS/Globals/GcTerrainGlobals.cs

```diff
@@ -47,33 +45,105 @@ public class GcTerrainGlobals : NMSTemplate
         /* 0x098 */ public NMSString0x10[] MiningSubstanceBiome;
-        [NMS(Size = 0x4, EnumType = typeof(GcGalaxyStarTypes.GalaxyStarTypeEnum))]
-        /* 0x198 */ public NMSString0x10[] MiningSubstanceStar;
-        [NMS(Size = 0x4, EnumType = typeof(GcGalaxyStarTypes.GalaxyStarTypeEnum))]
-        /* 0x1D8 */ public NMSString0x10[] MiningSubstanceStarExtreme;
-        [NMS(Size = 0x5)]
-        /* 0x218 */ public NMSString0x10[] MiningSubstanceRare;
-        /* 0x268 */ public float TextureBlendScale0;
-        /* 0x26C */ public float TextureBlendScale1;
-        /* 0x270 */ public float TextureBlendScale2;
-        /* 0x274 */ public float TextureBlendOffset;
-        /* 0x278 */ public GcTerrainOverlayColours HueOverlay;
-        /* 0x290 */ public GcTerrainOverlayColours SaturationOverlay;
-        /* 0x2A8 */ public GcTerrainOverlayColours ValueOverlay;
-        /* 0x2C0 */ public GcTerrainEditing TerrainEditing;
-        /* 0x358 */ public float TerrainUndoCubesRange;
-        /* 0x35C */ public float TerrainUndoCubesAlpha;
-        /* 0x360 */ public float TerrainUndoCubesNoiseThreshold;
-        /* 0x364 */ public float TerrainUndoCubesNoiseFactor;
-        /* 0x368 */ public float TerrainUndoFadeDepthConstant;
-        /* 0x36C */ public float TerrainUndoFadeDepthScalar;
-        /* 0x370 */ public bool DebugRegionHotspots;
+        /* 0x198 */ public List<NMSString0x10> MiningSubstanceStar;
+        /* 0x1A8 */ public List<NMSString0x10> MiningSubstanceStarExtreme;
+        /* 0x1B8 */ public List<NMSString0x10> MiningSubstanceRare;
+        /* 0x1C8 */ public float TextureBlendScale0;
+        /* 0x1CC */ public float TextureBlendScale1;
+        /* 0x1D0 */ public float TextureBlendScale2;
+        /* 0x1D4 */ public float TextureBlendOffset;
+        /* 0x1D8 */ public GcTerrainOverlayColours HueOverlay;
+        /* 0x1F0 */ public GcTerrainOverlayColours SaturationOverlay;
+        /* 0x208 */ public GcTerrainOverlayColours ValueOverlay;
+        /* 0x220 */ public GcTerrainEditing TerrainEditing;
+        /* 0x2B8 */ public float TerrainUndoCubesRange;
+        /* 0x2BC */ public float TerrainUndoCubesAlpha;
+        /* 0x2C0 */ public float TerrainUndoCubesNoiseThreshold;
+        /* 0x2C4 */ public float TerrainUndoCubesNoiseFactor;
+        /* 0x2C8 */ public float TerrainUndoFadeDepthConstant;
+        /* 0x2CC */ public float TerrainUndoFadeDepthScalar;
+        /* 0x2D0 */ public bool DebugRegionHotspots;
         [NMS(Size = 0x80)]
-        /* 0x371 */ public string RegionHotspotsTable;
-        [NMS(Size = 0x3, Ignore = true)]
-        /* 0x3F1 */ public byte[] Padding3F1;
-        /* 0x3F4 */ public int TerrainPrimeIndexStart;
-        [NMS(Size = 0x8, Ignore = true)]
-        /* 0x3F8 */ public byte[] EndPadding;
+        /* 0x2D1 */ public string RegionHotspotsTable;
+        /* 0x354 */ public int TerrainPrimeIndexStart;
     }
 }
+//using libMBIN.NMS.Toolkit;
```

Comment:
why is this all commented out? Is this the old code or...?

Reply:
yes, old code.  Should have removed it.  Will do in next batch